frontend/src/services/fetch/fetchers/applicationFetcher.ts

```diff
@@ -0,0 +1,21 @@
+import { APIResponse } from "src/types/apiResponseTypes";
+
+import { fetchApplications } from "./fetchers";
+
+interface applicationStartResponse extends APIResponse {
+  data: {
+    application_id: string;
+  };
+}
+
+// make call from server to API to start an application
+export const handleStartApplication = async (
+  competitionID: string,
+): Promise<applicationStartResponse> => {
+  const response = await fetchApplications({
+    subPath: `start`,
```

Comment:
is this always going to be a static value? If so we can put it in the namespace on the config

Reply:
There are a number of other application endpoints coming in upcoming tickets.